.drone.yml

```diff
@@ -28,31 +28,33 @@ steps:
     event:
     - tag
 
-- name: push
-  image: rancher/hardened-build-base:v1.15.8b5
+- name: manifest
+  image: golang:1.16
   volumes:
   - name: dockersock
     path: /var/run
   commands:
-  - docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
-  - TAG=${DRONE_TAG} make image-push
-  environment:
-    DOCKER_USERNAME:
-      from_secret: docker_username
-    DOCKER_PASSWORD:
-      from_secret: docker_password
+  - TAG=${DRONE_TAG} make image-manifest
   when:
     event:
     - tag
 
-- name: manifest
-  image: rancher/hardened-build-base:v1.15.8b5
-  volumes:
-  - name: dockersock
-    path: /var/run
-  commands:
-  - TAG=${DRONE_TAG} make image-manifest
+- name: docker-publish
+  image: plugins/docker
+  settings:
+    dockerfile: Dockerfile
+    password:
+      from_secret: docker_password
+    username:
+      from_secret: docker_username
+    repo: "rancher/gke-operator"
+    tag: "${DRONE_TAG}"
   when:
+    instance:
+    - drone-publish.rancher.io
+    ref:
+    - refs/head/master
+    - refs/tags/*
```

Comment:
This looks right. But I think to. use plugins/docker you have to also build from it: https://github.com/rancher/istio-installer/blob/master/.drone.yml#L7-L14

Reply:
Here is more advanced example: https://github.com/rancher/kubectl/blob/1e9a06830edeb3830b21770888ba778ae88dcd74/.drone.yml#L67-L86